modules/serverless-event-transformation-deployment-patterns.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// serverless/eventing/serverless-event-transformation.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="serverless-event-transformation-deployment-patterns_{context}"]
+= Deployment patterns for event transformation
+
+You can use `EventTransform` at different points in your event flow depending on your architecture needs. The following patterns are supported:
+
+[id="serverless-event-transformation-source-to-broker_{context}"]
+== Source to Broker event transformation
+
+You can apply an `EventTransform` resource between a Source and a Broker to modify events before they are delivered to the Broker. This pattern ensures that only normalized or enriched events enter the Broker for downstream processing. 
+
+You can transform events emitted by an `ApiServerSource` before sending them to the default Broker as shown in the following example: 
```

Comment:
Perhaps excplicitly saying that the are routed to the EventTransformer, which than transforms and routes them to the broker?

Reply:
I will make the same here as well.

Reply:
I have updated and refined the description.